src/libs/Pusher/pusher.ts

```diff
@@ -307,39 +300,31 @@ function isSubscribed(channelName) {
 
 /**
  * Sends an event over a specific event/channel in pusher.
- *
- * @param {String} channelName
- * @param {String} eventName
- * @param {Object} payload
  */
-function sendEvent(channelName, eventName, payload) {
+function sendEvent<T>(channelName: string, eventName: PusherEventName, payload: T) {
```

Comment:
What is the purpose of generic type here? We need to better type `payload` or at least limit the generic type so it extends `Record`.